CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
-Without this, any version satisfies the dependency. That means unless someone upgrades the extension outside of their process or incidentally install some package that uses newer version explicitly set, it will not automatically upgrade to the latest version.
-1. When creating a dependency **for virtual package, use exact version** of the dependent package (.install or .portable) which should be the same as that of virtual package.
-1. When taking a dependency on **anything else, specify minimum or exact version**.
+All packages in this repository are expected to pass all Requirement, Guideline and Suggestions checks on the Chocolatey Community Repository when it is possible.
+Additionally, all packages are also expected to following the additional Conventions / Guidelines we have outlined in this section.
 
-### 1.1.11 Provide uninstaller only if needed
+### Manual or automatic
 
-Packages should have uninstaller if auto uninstall doesn't work.
+When working on packages in this repository, a decision must be made whether the package should be an automatic or manual package.
+In most cases it is preferable that a package is made automatic, as these can then be updated without doing manual changes when there is a new release of the
+software that is being packaged.
+The only time an automatic package should be considered to be a manual package is if the underlying software that is being packaged has not been updated for 3 or more years.
+New packages contributed/migrated to this repository should always be created as automatic packages.
 
-### 1.1.12 Ensure compatibility with PowerShell v2+
+All automatic packages are expected to make use of the [AU](https://github.com/chocolatey-community/chocolatey-au) module, and work in Windows PowerShell 5.x, PowerShell Core is not supported.
 
-Chocolatey supports PowerShell v2+ because it supports Windows 7+/Windows Server 2008+ (Windows Server 2003 is supported currently, but is deprecated and expected to be removed in a future version). Due to that, PowerShell v2 is expected to work when you run anything here.
+The following metadata, and scripts conventions are for automatic packages, but manual packages are expected to follow the same rules when possible.
 
-For a reference of which cmdlets are available in PowerShell v2, please see the following link: http://adamringenberg.com/powershell2/
+### Package types
 
-### 1.1.13 Chocolatey compatibility
+### Embedded packages
 
-Packages are expected to be compatible with the Chocolatey version available 12 months ago.
-If a later version of Chocolatey is required, then a dependency on the earliest version compatible with the package must be added.
+_Embedded_ packages include the packaged software directly in the nupkg archive instead of downloading it. Only tools that allow redistribution in their license can be embedded and such packages must include two additional files in the directory `legal` - `VERIFICATION.txt` and `License.txt`.
```

Comment:
```suggestion
Embedded packages include the packaged software directly in the nupkg archive instead of downloading it. Only tools that allow redistribution in their license can be embedded and such packages must include two additional files in the `legal` directory within the source folder and shipped package - `VERIFICATION.txt` and `LICENSE.txt`.
```

Reply:
I think keeping the italic is still valid here, especially since we aren't removing it from the other sections (Sem-Embedded, Remote, Etc)

```suggestion
_Embedded_ packages include the packaged software directly in the nupkg archive instead of downloading it. Only tools that allow redistribution in their license can be embedded and such packages must include two additional files in the `legal` directory within the source folder and shipped package - `VERIFICATION.txt` and `LICENSE.txt`.
```

Reply:
Sure! Possibly at the time I hadn't got down to those lines yet, and thought it was an unusual highlight. Makes more sense with that context!